narwhals/dataframe.py

```diff
@@ -107,7 +107,8 @@ def with_row_index(self, name: str = "index") -> Self:
             self._compliant_frame.with_row_index(name),
         )
 
-    def drop_nulls(self: Self, subset: str | list[str] | None = None) -> Self:
+    def drop_nulls(self: Self, subset: str | list[str] | None) -> Self:
```

Comment:
wait I think it's fine for `narwhals.DataFrame.drop_nulls` to default to `None`? so users can just call `df.drop_nulls()`

Reply:
This is the `BaseFrame` implementation, the default still remains in `DataFrame` and `LazyFrame` and are passed along

Reply:
oops sorry! all good then 🙌